sql/1.145/maintainer_gh.pl

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env perl
+
+use FindBin;
+use lib "$FindBin::Dir/../../lib";
+use strict;
+use JSON;
+use DDGC;
+
+my $d  = DDGC->new;
+my $ias = $d->rs('InstantAnswer');
+
+while (my $ia = $ias->next) {
+    my $maintainer = $ia->maintainer;
+    my %updated_maintainer;
+    my $duckco_user = $d->rs('User')->find({username => $maintainer});
```

Comment:
Is there *any* chance the maintainer is not null but not found as a username?  It would essentially get left alone and cause errors.